editor/editor_help.cpp

```diff
@@ -915,16 +915,13 @@ void EditorHelp::_update_doc() {
 	// Properties overview
 	HashSet<String> skip_methods;
 
-	bool has_properties = cd.properties.size() != 0;
-	if (cd.is_script_doc) {
-		has_properties = false;
-		for (int i = 0; i < cd.properties.size(); i++) {
-			if (cd.properties[i].name.begins_with("_") && cd.properties[i].description.strip_edges().is_empty()) {
-				continue;
-			}
-			has_properties = true;
-			break;
+	bool has_properties = false;
+	for (const DocData::PropertyDoc &prop : cd.properties) {
+		if (prop.overridden || (cd.is_script_doc && prop.name.begins_with("_") && prop.description.strip_edges().is_empty())) {
+			continue;
 		}
+		has_properties = true;
+		break;
```

Comment:
I think this:
```suggestion
	bool has_properties = false;
	bool has_property_descriptions = false;
	for (const DocData::PropertyDoc &prop : cd.properties) {
		if (cd.is_script_doc && prop.name.begins_with("_") && prop.description.strip_edges().is_empty()) {
			continue;
		}
		has_properties = true;
		if (!prop.overridden) {
			has_property_descriptions = true;
			break;
		}
```
+changing:
https://github.com/godotengine/godot/blob/fb10f45efe52e330496e325b6b7efd0e52f769b0/editor/editor_help.cpp#L1603-L1604
to check `has_property_descriptions` should do the thing.